src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactory.java

```diff
@@ -225,26 +225,33 @@ protected VariantKeyFields normalizeLeftAlign(String chromosome, int position, S
 
     protected void parseSplitSampleData(Variant variant, String fileId, String studyId, String[] fields,
                                         String[] alternateAlleles, String[] secondaryAlternates,
-                                        int alternateAlleleIdx) throws NonStandardCompliantSampleField {
+                                        int alternateAlleleIdx) throws NonStandardCompliantSampleField, NotAVariantException {
         String[] formatFields = variant.getSourceEntry(fileId, studyId).getFormat().split(":");
 
         for (int i = 9; i < fields.length; i++) {
             Map<String, String> map = new TreeMap<>();
-
+            int countreferencegenotypes=0;
```

Comment:
regarding the code style, use camelCase, spaces around words and around braces, look at other parts of the code for examples